doc/source/api_changes.rst

```diff
@@ -17,6 +17,8 @@ Version 0.13
   0.15, will be set to `True`.
 - Parameters ``ntiles_x``, ``ntiles_y`` have been removed from
   ``skimage.exposure.equalize_adapthist``.
+- The ``freeimage`` io plugin is no longer supported, and will use ``imageio``
```

Comment:
There is something I misunderstand. You removed a task attributed for 0.14 and you added an item for 0.13. When do we want to remove freeimage?

We need to agree on that and then, be consistent.